server/webdriver/tests/standalone/sanity.py

```diff
@@ -223,6 +223,16 @@ def explore_landing(self, page: WebPage):
     page.title = self.driver.title if page.title is None else page.title
 
     # queries
+    try:
+      WebDriverWait(self.driver, 3).until(
```

Comment:
Extract out the 3 second timeout to a variable

Reply:
done!